src/routes/jira/connect/enterprise/jira-connect-enterprise-delete.ts

```diff
@@ -1,15 +1,23 @@
 import { NextFunction, Request, Response } from "express";
 import { GitHubServerApp } from "models/github-server-app";
+import { Installation } from "models/installation";
 
 export const JiraConnectEnterpriseDelete = async (
 	req: Request,
 	res: Response,
 	next: NextFunction
 ): Promise<void> => {
 	try {
+
 		req.log.debug("Received Jira Connect Enterprise Server DELETE request");
 
-		await GitHubServerApp.uninstallServer(req.body.serverUrl);
+		const { jiraHost }  = res.locals;
+		const installation = await Installation.getForHost(jiraHost);
```

Comment:
Don't we have it in res.locals.installation already?

Reply:
I believe we do :)

Reply:
> Don't we have it in res.locals.installation already?

Maybe, it has the following in jwt middleware
```
	// TODO: Probably not the best place to set things globally
	res.locals.installation = installation;
```

Should I use it?

Reply:
I believe yes :) cause it was driven by JWT, which means we could trust that? At least this is my thinking

Reply:
Done. :)